index.js

```diff
@@ -137,19 +138,32 @@ async function handleAlert(location, alertType) {
 
     if (currentSession === null || currentTime - currentSession.updatedAt >= helpers.getEnvVar('SESSION_RESET_THRESHOLD')) {
       const newSession = await db.createSession(location.locationid, location.responderPhoneNumber, alertType, client)
-      const alertInfo = {
-        sessionId: newSession.id,
-        toPhoneNumber: location.responderPhoneNumber,
-        fromPhoneNumber: location.twilioNumber,
-        message: `This is a ${alertTypeDisplayName} alert. Please check on the bathroom at ${location.displayName}. Please respond with 'ok' once you have checked on it.`,
-        reminderTimeoutMillis: location.reminderTimer,
-        fallbackTimeoutMillis: location.fallbackTimer,
-        reminderMessage: `This is a reminder to check on the bathroom`,
-        fallbackMessage: `An alert to check on the bathroom at ${location.displayName} was not responded to. Please check on it`,
-        fallbackToPhoneNumbers: location.fallbackNumbers,
-        fallbackFromPhoneNumber: location.twilioNumber,
+      
+      if (location.siren_particle_id !== null) {
+        try{
+          await axios.post('https://api.particle.io/v1/devices/${location.siren_particle_id}/start-siren', {
```

Comment:
So unfortunately `location` doesn't automatically get the `siren_particle_id` value just because it's been added to the DB. You'll need to update the `Location.js` class to add it. And then in `db.js`, there is a function called `createLocationFromRow` that will need to pass the `siren_particle_id` as a parameter to the contructor that you changed in `Location.js`. Then you'll be able to use it here.